pypowsybl/network.py

```diff
@@ -4123,6 +4156,23 @@ def remove_extensions(self, extension_name: str, ids: _Union[str, _List[str]]) -
             ids = [ids]
         _pp.remove_extensions(self._handle, extension_name, ids)
 
+    def get_slack(self) -> _DataFrame:
```

Comment:
After more thoughts, I would propose to remove for now those additional network methods:
as such they have no added value compared to the extension.
It could have some added value if, for example, it was also taking care of removing existing slack buses, but let's postpone this kind of addition until the need arises.